discovery/gossiper.go

```diff
@@ -1344,6 +1344,13 @@ func (d *AuthenticatedGossiper) retransmitStaleChannels() error {
 
 	var signedUpdates []lnwire.Message
 	for _, chanToUpdate := range edgesToUpdate {
+		// Ensure that the max HTLC field is set. If it isn't, set it to the
+		// capacity of the channel.
+		if chanToUpdate.edge.MaxHTLC == 0 {
```

Comment:
I don't think we have to do this, as we will just send out a `channel_update` with the flag set to 0.

Reply:
Fixed. FWIW it's worth it was requested in this [comment](https://github.com/lightningnetwork/lnd/pull/2296#issuecomment-452174100)